lib/common.ts

```diff
@@ -134,13 +134,28 @@ export function runProgram(
 ) {
   const transform = new ReadlineTransform({ skipEmpty: true });
   const proc = spawn(command, arguments_, options);
+  let recentMessage: string;
+  let errorMessage: string;
   proc.stdout.setEncoding("utf8");
   proc.stdout
-    .pipe(transform)
-    // TODO: Review this JSONStream.parse()
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    .pipe(JSONStream.parse())
+    .pipe(
+      transform.on("error", (error) => {
+        throw error;
```

Comment:
This wasn't strictly necessary, but it did convert an error location from within `node:stream` to this userland function.